### PR TITLE
Add 2023.06 release notes to .metainfo.xml

### PR DESCRIPTION
**Pull request by Ferenc-**

### CODE REVIEW

**Comment by biodranik** [comment]
• New OpenStreetMap data as of June 2, 2023
• GPX import support
• Fixed disappearing bookmark lists when wrong characters were saved in a bookmark name or description
• Improved car and bicycle routing
• Fixed bookmarks dialog color in the dark mode

Styles:
• Fixed funicular icon
• Changed desert color in the dark mode
• Added Fish Pass
• New Icon for Parcel Locker
• Fixed icon for northernmost Munich underground station

Translations:
• Translations for opening hours picker
• Corrected Basque, Belarusian, British, English, and Ukrainian translations

**Comment by Ferenc-** [comment]
Done

File: packaging/app.organicmaps.desktop.metainfo.xml [hunk 1]
```diff
@@ -113,6 +113,31 @@
...
+         <ul>
+           <li>Fixed funicular icon</li>
+           <li>Changed desert color in the dark mode</li>
+           <li>Added Fish Pass</li>
+           <li>New Icon for Parcel Locker</li>
+           <li>Fixed icon for northernmost Munich underground station</li>
+         </ul>
+       <p>Translations:</p>
+         <ul>
+           <li>Translations for opening hours picker</li>
```
Comment (biodranik) [inline packaging/app.organicmaps.desktop.metainfo.xml:136]: Uuups, this line should be removed, it's for Android-only.

Comment (Ferenc-) [reply packaging/app.organicmaps.desktop.metainfo.xml]: Removed.

**Review by biodranik — APPROVED** [review]

**Pull request merged**